pyrit/prompt_converter/ascii_art_converter.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pyrit.prompt_converter import PromptConverter
+from art import text2art
+
+
+class AsciiArtConverter(PromptConverter):
+    """Converts a string to ASCII art"""
+
+    def __init__(self, font="rand"):
+        self.font_value = font
+
+    def convert(self, prompts: list[str]) -> list[str]:
+        """
+        Converter that uses art to convert strings to ASCII art
```

Comment:
```suggestion
        Converter that uses art to convert strings to ASCII art.
```
Nit: adding this since the new line won't show if we generate docs from this, so it'll connect the two sentences without full stop.